api/osrd_infra/migrations/0018_rollingstockcompoundimage_rollingstocklivery_and_more.py

```diff
@@ -0,0 +1,148 @@
+# Generated by Django 4.1.5 on 2023-01-19 13:03
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("osrd_infra", "0017_remove_lpvpanellayer_obj_lpvpanellayer_obj_id"),
+    ]
+
+    operations = [
+            migrations.CreateModel(
+                name="RollingStockCompoundImage",
+                fields=[
+                    ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                    ("image", models.BinaryField()),
+                ],
+                options={
+                    "verbose_name_plural": "rolling stock compound images",
+                    "db_table": "osrd_infra_rollingstockimage",
+                    "managed": False,
+                },
+            ),
+            migrations.CreateModel(
+                name="RollingStockLivery",
+                fields=[
+                    ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                    ("name", models.CharField(default="default", help_text="Name of the livery", max_length=255)),
+                ],
+                options={
+                    "verbose_name_plural": "rolling stocks liveries",
+                },
+            ),
+            migrations.CreateModel(
+                name="RollingStockSeparatedImage",
+                fields=[
+                    ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                    ("image", models.BinaryField()),
+                    ("order", models.IntegerField(default=0, help_text="Position of this image in its livery")),
+                ],
+                options={
+                    "verbose_name_plural": "rolling stock separated images",
+                    "db_table": "osrd_infra_rollingstockimage",
+                },
+            ),
+            # rolling_stock_images: set order default to 0
+            migrations.RunSQL(
+                sql=[(f"""ALTER TABLE osrd_infra_rollingstockimage
+                        ALTER COLUMN "order" SET DEFAULT 0;
+                    """)],
+                reverse_sql=[(f"""ALTER TABLE osrd_infra_rollingstockimage
+                        ALTER COLUMN "order" DROP DEFAULT;
+                    """)],
+            ),
+            # compound_image: OneToOne dans RollingStockLivery vers RollingStockImage
```

Comment:
```suggestion
            # compound_image: OneToOne in RollingStockLivery with RollingStockImage
```

Reply:
There are others in French below :smile:

Reply:
Oupsi, merci !!